testing/run_tests.py

```diff
@@ -247,7 +257,7 @@ def RunDartTest(build_dir, dart_file, verbose_dart_snapshot, multithreaded, enab
     threading = 'single-threaded'
 
   print("Running test '%s' using 'flutter_tester' (%s)" % (kernel_file_name, threading))
-  RunEngineExecutable(build_dir, 'flutter_tester', None, command_args)
+  RunEngineExecutable(build_dir, 'flutter_tester', None, command_args, forbidden_output=['[ERROR'])
```

Comment:
What other errors are emitted? We should just patch the tester to provide necessary stubs.

Reply:
This is asserting the output doesn't contain an error log from the shell